variant/validators/polypeptide_sequence_variant_base.py

```diff
@@ -146,6 +144,7 @@ def get_valid_invalid_results(self, classification_tokens, transcripts,
         :param list results: A list to store validation result objects
         :param list gene_tokens: List of GeneMatchTokens
         """
+        valid_alleles = list()
```

Comment:
On line 138 (beginning of the docstring) do you still need the `noqa` parameter? From eyeballing it, I think the way it is now shouldn't throw an error

IMO you could also tweak the wording a bit on the errors/results param descriptions to make it clearer that the object could potentially be altered by the function (since it sort of implicitly uses them as return types instead of explicitly returning something)